src/SDSS.jl

```diff
@@ -1,21 +1,26 @@
 module SDSS
 
+VERSION < v"0.4.0-dev" && using Docile
 using CelesteTypes
+using FITSIO
+using WCSLIB
+using DataFrames
+using Grid
+using GaussianMixtures
 
 # FITSIO v0.6.0 removed some of the lower-level functions used here.
 # In particular, I think wcslib may still need this.
```

Comment:
I'm being a little lazy here -- I don't yet know how WCSLIB works, and I don't know whether / why you need to pass the output of FITSIO.Libcfitsio.fits_hdr2str to wcspih.  It's the hdr2str function that needs the lower-level FITSIO stuff.  If you know offhand how to get around this, I'd be happy to change it, otherwise, I'll just fix it when I dig into the rest of the WCS stuff.